onset_maproom/layout.py

```diff
@@ -410,6 +398,7 @@ def results_layout():
                     dbc.Spinner(dcc.Graph(id="probExceed_cess")),
                 ],
                 label="Cessation Date",
+                style={"display": CONFIG["isonoff_cess_date_hist"]}
```

Comment:
In the browser dev tools I can see that the cessation date graph is being fetched from the backend, even though it will never be displayed. Let's use a different mechanism for hiding it (and hide the tab while we're at it).

The first thing that comes to mind is to give the `dbc.Tabs` element no children in the layout, and set the children in a callback.

Reply:
I have a solution in the last couple commits. It inflates a bit the repeat codes for trying and alerts for graphs that we said we wanted to rationalize but didn't get to.
I am not sure I need to create alert figures for the cast no tab... Can I just return None?

Reply:
I'm not sure if returning None will work in this particular context. Try returning None or `{}`. If you really have to return a Figure, you can probably do it in one line instead of ten--skip the call to `add_annotation` and just return a Figure object with no content.

Reply:
Aside from that simplification, this approach seems fine. Another approach would be to generate the tab(s) in a callback rather than in the static layout, so that in the Senegal case the cessation tab is simply not there, rather than being present with display: none. I don't know offhand what the relative advantages/disadvantages would be.

Reply:
yeah... if this maproom is going to get more complex in the future, with more options that can be triggered on/off, maybe it's easier to manage the layout if all possible elements are in there. Don't know... thinking out loud...